src/CalcManager/CEngine/History.cpp

```diff
@@ -2,18 +2,41 @@
 // Licensed under the MIT License.
 
 #include "pch.h"
-#pragma once
 #include "Header Files/CalcEngine.h"
 #include "Command.h"
 #include "CalculatorVector.h"
 #include "ExpressionCommand.h"
-#include "CalcException.h"
 
 constexpr int ASCII_0 = 48;
 
 using namespace std;
 using namespace CalcEngine;
 
+class CalcException : public std::exception
```

Comment:
On second look, this class isn't needed.  It doesn't override `std::exception::what` and nowhere in the codebase explicitly catches a `CalcException`.

Reply:
done